frontend/src/store/my-loans/mutations.ts

```diff
@@ -9,4 +9,30 @@ export const mutations = {
   'REQUEST_MY_LOANS': function (state): void {
     state.isLoading = true;
   },
+  'START_TRANSFERING_COLLATERAL': function (state: MyLoansStateT, { shortId }): void {
+    state.myLoansList = state.myLoansList.map(loan => {
+      if (loan.shortId == shortId) {
+        return {
+          ...loan,
+          isTransferingCollateral: true
+        }
+      } else {
+        return loan;
+      }
+    })
+  },
+  'COLLATERAL_TRANSFERED': function (state: MyLoansStateT, { shortId }): void {
```

Comment:
This mutation should maybe receive an updated loan from the library, and replace it fully instead? I'm not sure.

Reply:
Ack.